### PR TITLE
Add translation to the update manager for pt-BR.

### PR DESCRIPTION
**Pull request by boaglio**
Update for pt_BR translation.

### Testing 

Running local instance to check translations. 

image

### Proposed changelog entries

Add translation to the update manager for pt-BR.

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7255"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/UpdateCenter/index_pt_BR.properties [hunk 1]
```diff
@@ -20,6 +20,6 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Update\ Center=Central de atualização
-# Once the installation is completed, Jenkins needs to be restarted for changes to take effect.
-Installing\ Plugins/Upgrades=Instalando plugins/atualizando
+Plugin\ Manager=Central de atualização
+Download\ progress=Andamento do download
+
```
Comment (NotMyFault) [inline core/src/main/resources/hudson/model/UpdateCenter/index_pt_BR.properties:25]:
```suggestion
```

File: core/src/main/resources/lib/hudson/iconSize_pt_BR.properties [hunk 2]
```diff
@@ -21,6 +21,9 @@
 # THE SOFTWARE.
 
 Icon=Ícone
-L=L
-S=S
+
+S=P
+
+L=G
+
```
Comment (NotMyFault) [inline core/src/main/resources/lib/hudson/iconSize_pt_BR.properties:28]: No need to keep a spare line between the entries.

File: core/src/main/resources/hudson/model/UpdateCenter/index_pt_BR.properties (hunk 1 above)
Comment (boaglio) [reply core/src/main/resources/hudson/model/UpdateCenter/index_pt_BR.properties]: ok, I will fix it

File: core/src/main/resources/lib/hudson/iconSize_pt_BR.properties (hunk 2 above)
Comment (boaglio) [reply core/src/main/resources/lib/hudson/iconSize_pt_BR.properties]: ok, I will fix it

File: core/src/main/resources/lib/hudson/iconSize_pt_BR.properties [hunk 3]
```diff
@@ -21,6 +21,6 @@
 # THE SOFTWARE.
 
 Icon=Ícone
-L=L
-S=S
+S=P
+L=G
 M=M
```
Comment (basil) [inline core/src/main/resources/lib/hudson/iconSize_pt_BR.properties:26]: Note that [the English versions of these](https://github.com/jenkinsci/jenkins/blob/34d098dfaa5097e77b4de621a4d0af236432d2a5/core/src/main/resources/lib/hudson/iconSize.properties#L24-L26) are "Small", "Medium", and "Large", and [the rendering code](https://github.com/jenkinsci/jenkins/blob/34d098dfaa5097e77b4de621a4d0af236432d2a5/core/src/main/resources/lib/hudson/iconSize.jelly) displays the first letter on the page while displaying the full word as a tooltip. In its current form, this PR only displays the first letter on the page and as a tooltip, which is inconsistent with the English version. To be consistent, this should include the full word:

```suggestion
S=Pequeno
L=Grande
M=Médio
```

**Review by basil — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**